Makefile

```diff
@@ -59,7 +73,7 @@ l10n_deps: translationtool.phar
 translationtool.phar:
 	wget https://github.com/nextcloud/docker-ci/raw/master/translations/translationtool/translationtool.phar -O translationtool.phar
 
-deps: download_models composer javascript_deps l10n_deps
+deps: download_models composer javascript_deps l10n_deps handlebars
```

Comment:
Consider putting handlebars first, so it blows before it execute those "slow" stuff, like "download_models"

Reply:
I put it at the end so that the message is seen. As it does not generate an error, it does not stop downloading the rest of the dependencies...

Reply:
makes sense, keep it:)